curriculum/challenges/english/06-quality-assurance/quality-assurance-and-testing-with-chai/use-regular-expressions-to-test-a-string.english.md

```diff
@@ -13,8 +13,8 @@ As a reminder, this project is being built upon the following starter project on
 
 ## Instructions
 <section id='instructions'>
-Use <code>assert.match()</code> to make the tests pass. 
-
+Use <code>assert.match()</code> or <code>assert.notMatch()</code> to make the tests pass.
```

Comment:
```suggestion
Use <code>assert.match()</code> and <code>assert.notMatch()</code> to make the tests pass.
```

Reply:
I think "or" is fine, since the user would not use both `match` and `noMatch` in the same test.